x-pack/legacy/plugins/telemetry/public/services/telemetry_opt_in.test.mocks.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { injectedMetadataServiceMock } from '../../../../../../src/core/public/mocks';
```

Comment:
Regarding my earlier comment about relative imports going out of OSS into X-Pack, that might not apply to test code